gapil/semantic/type.go

```diff
@@ -512,6 +512,30 @@ func UnsignedInteger(size int32) Type {
 	}
 }
 
+// AsUint64 converts a sized integer value to a uint64.
+func AsUint64(val Expression) uint64 {
```

Comment:
Please migrate the other switch statement over to call this function, and return a `(uint64, bool)` pair, where the second is a success or failure.

Reply:
I did, I think the PR didn't update to show the other file.  Changed to `(uint64, bool)` now 👍